AppointmentReminders.Web/Controllers/AppointmentsController.cs

```diff
@@ -48,10 +94,34 @@ public ActionResult Edit(int? id)
                 return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
             }
 
-            // Find participant
+            var appointment = _repository.FindById(id.Value);
+            if (appointment == null)
+            {
+                return HttpNotFound();
+            }
 
-            return View(new Appointment());
+            ViewBag.Timezones = Timezones;
+            return View(appointment);
+        }
+
+        // POST: /Appointments/Edit/5
+        [HttpPost]
+        public ActionResult Edit([Bind(Include = "ID,Name,PhoneNumber,Time,Timezone")] Appointment appointment)
+        {
+            if (ModelState.IsValid)
+            {
+                _repository.Update(appointment);
+                return RedirectToAction("Details", new { id = appointment.Id });
+            }
+            return View(appointment);
+        }
 
+        // GET: Appointments/Delete/5
```

Comment:
Can't we use this?
https://msdn.microsoft.com/en-us/library/system.web.mvc.htmlhelper.httpmethodoverride(VS.100).aspx

On Fri, Jul 17, 2015 at 11:07 AM, Agustin Camino <notifications@github.com>
wrote:

> In AppointmentReminders.Web/Controllers/AppointmentsController.cs
> <https://github.com/TwilioDevEd/appointment-reminders-csharp/pull/1#discussion_r34904041>
> :
>
> >
> > +        // GET: Appointments/Delete/5
>
> The alternative is to use another helper and do the Delete action through
> an Ajax call.
> More explicitly we can use Ajax Helper
> <https://msdn.microsoft.com/en-us/library/system.web.mvc.ajax.ajaxextensions.actionlink(v=vs.118).aspx>
> .
>
> I'm not sure if this means to add artificial complexity to the
> application. WDYT?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TwilioDevEd/appointment-reminders-csharp/pull/1/files#r34904041>
> .
>